stats/cloud/collector.go

```diff
@@ -284,6 +284,7 @@ func (c *Collector) Collect(sampleContainers []stats.SampleContainer) {
 
 			if sc.FullIteration {
 				values[metrics.IterationDuration.Name] = stats.D(sc.EndTime.Sub(sc.StartTime))
+				values[metrics.Iterations.Name] = 1
```

Comment:
This seems a bit silly since it's not processing the actual sample, but because it should always be `1`(?) I figured it's easier to just assign it.

It also didn't make sense to create a new `NetTrail.Iterations` field for this, but let me know if this can be improved.

Reply:
I think this is fine for now. Even we start to aggregate them we will be aggregating them here(for now .. probably), so it wouldn't matter. Even we aggregate outside the output, it's still a small change ;).